components/Footer.tsx

```diff
@@ -0,0 +1,8 @@
+function Footer() {
+    return (
+        <footer className="footer">
+        <div className="content has-text-centered"></div>
+        </footer>
+    );
+}
+export default Footer;
```

Comment:
please don't use [default export ](https://blog.neufund.org/why-we-have-banned-default-exports-and-you-should-do-the-same-d51fdc2cf2ad?gi=14509d384636)